src/main/resources/application.conf

```diff
@@ -96,7 +96,7 @@ grothendieck {
 
       # Maximum number of reconnect attempts after the connection has been initiated.
       # After that, the connection will be dropped until its initiated again (eg. by peer discovery)
-      connect-max-retries = 1
+      connect-max-retries = 5
```

Comment:
maybe decrease it and increase connect-retry-delay ?  trying 5 times in 5 seconds intervals doesn't seem like good approach